stacker/actions/build.py

```diff
@@ -89,6 +89,10 @@ def resolve_parameters(parameters, blueprint, context, provider):
             values = value.split(",")
             for v in values:
                 v = v.strip()
+                # Skip parameters not referencing another stack
```

Comment:
I also wonder if the whole logic of how we parse the values should be pulled out into it's own function, so we don't have to update it in two places like you had to here.